hyper/contrib.py

```diff
@@ -51,8 +57,8 @@ def send(self, request, stream=False, **kwargs):
         Sends a HTTP message to the server.
         """
         parsed = urlparse(request.url)
-
-        conn = self.get_connection(parsed.hostname, parsed.port, parsed.scheme)
+        conn = self.get_connection(parsed.hostname, parsed.port, parsed.scheme,
+                                   cert=kwargs.get('cert'))
```

Comment:
Rather than do a `kwargs.get`, let's just add `cert` to our list of explicit kwargs.

Also, when the arguments no longer fit on one line I'd rather it look more like this:

```python
conn = self.get_connection(
    parsed.hostname,
    parsed.port,
    parsed.scheme,
    cert
)
```

This is just a silly stylistic thing, but still. =)